fyle_accounting_mappings/views.py

```diff
@@ -37,7 +37,8 @@ def post(self, request, *args, **kwargs):
 
             assert_valid(mapping_settings != [], 'Mapping settings not found')
 
-            mapping_settings = MappingSetting.bulk_upsert_mapping_setting(mapping_settings, self.kwargs['workspace_id'])
+            MappingSetting.bulk_upsert_mapping_setting(mapping_settings, self.kwargs['workspace_id'])
+            mapping_settings: MappingSetting = MappingSetting.objects.filter(workspace_id=self.kwargs['workspace_id'])
```

Comment:
I assume this might be needed in app, if you let know the exact use case in app, we can try to avoid this. This is slightly hacky.

Reply:
now we have two separate buttons in expense field section which submit two different forms, now in this when we submit one form we only get mapping setting of that particular form